src/world/FieldMessage.cpp

```diff
@@ -115,6 +116,10 @@ FieldLineSegment FieldMessage::getBottom_line(){
 FieldLineSegment FieldMessage::getLeft_line(){
   return left_line;
 }
+double FieldMessage::getLeftLineX() {
```

Comment:
Functionality like this is probably better to do in the field class itself.

Reply:
I do not agree, I want to use the FieldMessage class to compute direct things about the Field (such as sizes, line positions and coordinates) and the Field class itself to compute indirect things about the Field (whether a point is left of a line).